app/frontend/src/components/designer/FormSettings.vue

```diff
@@ -263,6 +263,40 @@
               </v-tooltip>
             </template>
           </v-checkbox>
+          <v-checkbox
+            class="my-0"
+            v-model="subscribe.enabled"
+            :disabled="idirUser === false || !isFormPublished"
+          >
+            <template #label>
+              <span
+                style="max-width: 80%"
+                v-html="$t('trans.formSettings.allowEventSubscription')"
+              />
+              <v-tooltip bottom close-delay="2500">
+                <template v-slot:activator="{ on, attrs }">
+                  <font-awesome-icon
+                    icon="fa-solid fa-flask"
+                    color="primary"
+                    class="ml-3"
+                    v-bind="attrs"
+                    v-on="on"
+                  />
+                </template>
+                <span
+                  >{{ $t('trans.formSettings.experimental') }}
+                  <a
+                    :href="githubLinkCopyFromExistingFeature"
```

Comment:
Just so we don't forget - this will be updated to the new URL for the experimental feature.